pytest_splunk_addon/standard_lib/data_models/Malware.json

```diff
@@ -71,7 +103,7 @@
         },
         {
           "name": "vendor_product",
-          "type": "optional",
+          "type": "required",
```

Comment:
again, nice idea but products in the real world don't always have this. we can leave it and when it causes too many excepted failures we can revert it.

Reply:
We always know the product name, not necessarily extracted from the raw but as a static value.